db/seeds.rb

```diff
@@ -1,14 +1,13 @@
 # Seed the DB with an admin user and a few version records
 admin = User.find_or_create_by(admin: true) do |user|
-  password = SecureRandom.uuid
+  password = 'PASSWORD'
   user.email = 'seed-admin@example.com'
   user.password = password
   user.confirmed_at = Time.now
 
   edit_path = Rails.application.routes.url_helpers.edit_user_registration_path
   puts "\n\n------------------------------------------------------------"
   puts "Admin user created with e-mail: #{user.email} and password: #{password}"
-  puts "Point your browser to http://localhost:3000#{edit_path} to change it."
```

Comment:
I wonder if this message would still be useful.

Regardless, please either put this back or remove the line declaring `edit_path` above to resolve the Rubocop error.